metabase/metabase
# Dashboard Viz: Prevent 'Popover' on clicking 'View More'

Closes https://github.com/metabase/metabase/issues/42790

### Description

Changes with this PR should prevent popover on Expanding/Collapsing body in detail of Sample Database.

### How to verify

1. Go to Database -> Sample Database -> Review Table -> Detail Visualization 
2. Click on View more/less
3. On clicking, filter popup should not appear unless user explicitly clicks on the body text.

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
